addons/hr_timesheet_attendance/report/hr_timesheet_attendance_report.py

```diff
@@ -17,6 +17,7 @@ class TimesheetAttendance(models.Model):
 
     @api.model_cr
     def init(self):
+        self._cr.execute("DROP VIEW IF EXISTS hr_timesheet_attendance_report")
```

Comment:
`self._cr.execute("DROP VIEW IF EXISTS %s" % self._table)`

Reply:
You are right, I will correct it asap and notify you on the new PR.
Thanks a lot for your commitment.